testing/db-testkit/src/main/kotlin/net/corda/db/testkit/DbUtils.kt

```diff
@@ -4,105 +4,143 @@ import com.typesafe.config.Config
+import net.corda.db.core.SQLDataSourceFactory
+import net.corda.schema.configuration.DatabaseConfig
+
 
 object DbUtils {
+
     private const val POSTGRES_HOST_PROPERTY = "postgresHost"
     private const val POSTGRES_PORT_PROPERTY = "postgresPort"
+    private const val MSSQL_HOST_PROPERTY = "mssqlHost"
+    private const val MSSQL_PORT_PROPERTY = "mssqlPort"
 
-    private val logger: Logger = LoggerFactory.getLogger(this::class.java)
+    val isPostgres = System.getProperty(POSTGRES_PORT_PROPERTY).isNullOrBlank()
+    val isMSSQL = System.getProperty(MSSQL_PORT_PROPERTY).isNullOrBlank()
 
-    val isInMemory = System.getProperty(POSTGRES_PORT_PROPERTY).isNullOrBlank()
+    val isInMemory = System.getProperty(POSTGRES_PORT_PROPERTY).isNullOrBlank() && System.getProperty(
+        MSSQL_PORT_PROPERTY).isNullOrBlank()
 
-    /**
-     * Returns Postgres DB name
-     */
-    fun getPostgresDatabase() = getPropertyNonBlank("postgresDb", "postgres")
+    private val logger: Logger = LoggerFactory.getLogger(this::class.java)
 
-    /**
-     * Returns DB admin user
-     */
-    fun getAdminUser() = if (isInMemory) "sa" else getPropertyNonBlank("postgresUser", "postgres")
+    fun getDatabase() =
+        if (!isPostgres){
+            getPropertyNonBlank("postgresDb", "postgres")
+        }
+        else if (!isMSSQL){
+            getPropertyNonBlank("sqlDb", "sql")
+        }
+        else {
+            ""
+        }
 
-    /**
-     * Returns DB admin user's password
-     */
-    fun getAdminPassword() = if (isInMemory) "" else getPropertyNonBlank("postgresPassword", "password")
+    fun getAdminUser() =
+        if(!isPostgres){
+            getPropertyNonBlank("postgresUser","postgres")
+        }
+        else if(!isMSSQL){
+            getPropertyNonBlank("mssqlUser", "sa")
+        }
+        else{
+            "sa"
+        }
 
-    /**
-     * Get a Postgres EntityManager configuration if system properties set as necessary. Otherwise, falls back on
-     * in-memory implementation.
-     */
+    fun getAdminPassword() =
+        if(!isPostgres){
+            getPropertyNonBlank("postgresPassword", "password")
+        }
+        else if(!isMSSQL){
+            getPropertyNonBlank("mssqlPassword", "password")
+        }
+        else{
+            ""
+        }
     @Suppress("LongParameterList")
     fun getEntityManagerConfiguration(
-        inMemoryDbName: String,
+        inMemoryDbName : String,
         dbUser: String? = null,
         dbPassword: String? = null,
         schemaName: String? = null,
         createSchema: Boolean = false,
         showSql: Boolean = true,
         rewriteBatchedInserts: Boolean = false
-    ): EntityManagerConfiguration {
-        val port = System.getProperty(POSTGRES_PORT_PROPERTY)
-        return if (!port.isNullOrBlank()) {
-            val ds = createPostgresDataSource(dbUser, dbPassword, schemaName, createSchema, rewriteBatchedInserts)
+    ):EntityManagerConfiguration{
```

Comment:
```suggestion
    ):EntityManagerConfiguration {
```

Again, unnecessary (and incorrect) whitespace change

(This looks to be a recurring theme in the PR - please review all instances and revert where necessary)